src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -278,20 +279,20 @@ def _register_python_wheel_task(self, graph: DependencyGraph) -> Iterable[Depend
             ]
         return graph.register_import(entry_point.module)
 
-    def _register_spark_jar_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
+    def _register_spark_jar_task(self, _) -> Iterable[DependencyProblem]:
```

Comment:
Even if unused, shouldn't the hint still be there?

Reply:
nope